### PR TITLE
[CROSSDATA-124] [Mongo Datasource] Allow filter _id like objectID or another type

### PR DESCRIPTION
**Pull request by pmadrigal**

### CODE REVIEW

File: doc/src/site/sphinx/First_Steps.rst [hunk 1]
```diff
@@ -104,7 +104,10 @@ Configuration parameters
 +-----------------------------------------------+--------------------------------------------------------------------------------+-------------------------+
 | threadsAllowedToBlockForConnectionMultiplier  |   "5"                                                                          | No                      |
 +-----------------------------------------------+--------------------------------------------------------------------------------+-------------------------+
+| idasobjectid                                  |   "false"                                                                      | No                      |
++-----------------------------------------------+--------------------------------------------------------------------------------+-------------------------+
 
+**Note:** '_id' field is autogenerated in MongoDB, so by default, you can filter it as String. If you need a custom '_id', you have to set 'objectid' property to "false" like in the above table.
```
Comment (ccaballe) [inline doc/src/site/sphinx/First_Steps.rst]: property is 'idasobjectid' instead of 'objectid'

**Comment by ccaballe** [comment]
+1

File: spark-mongodb/src/main/scala/com/stratio/datasource/mongodb/reader/MongodbReader.scala [hunk 2]
```diff
@@ -141,6 +141,25 @@ class MongodbReader(
...
+   * @param attribute Name of the file
+   * @param value Value for the attribute
+   * @return The value in the correct data type
+   */
+  private def checkObjectID(attribute: String, value: Any) : Any = attribute  match {
+    case "_id" if idAsObjectId => new ObjectId(value.toString)
+    case _ => value
+  }
+
+  private def idAsObjectId: Boolean =
```
Comment (pfcoperez) [inline spark-mongodb/src/main/scala/com/stratio/datasource/mongodb/reader/MongodbReader.scala]: No need of `if`: `private def idAsObjectId: Boolean = config.getOrElse[String](MongodbConfig.IdAsObjectId, MongodbConfig.DefaultIdAsObjectId).equalsIgnoreCase("true")

**Comment by pfcoperez** [comment]
+1

**Pull request merged**